tests/staticfiles_tests/test_storage.py

```diff
@@ -299,10 +300,20 @@ def test_corrupt_intermediate_files(self):
             self.hashed_file_path('cached/styles.css')
 
 
-@override_settings(
-    STATICFILES_STORAGE='staticfiles_tests.storage.ExtraPatternsCachedStaticFilesStorage',
-)
-class TestExtraPatternsCachedStorage(CollectionTestCase):
+class TestCachedStaticFilesStorageDeprecation(SimpleTestCase):
+    def test_warning(self):
+        from django.contrib.staticfiles.storage import CachedStaticFilesStorage
+        from django.utils.deprecation import RemovedInDjango31Warning
```

Comment:
I guess these imports were moved here for easier removal once the deprecation period ends?

Reply:
Yes